### PR TITLE
update Manila component installation (SCRD-8940)

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

File: xml/installation-ardana-manila.xml [hunk 1]
```diff
@@ -92,69 +127,285 @@ the changes to take effect:
...
+     Share Server management creates Storage Virtual Machines (SVM), thus SVM
+     administrator privileges are insufficient.
+    </para>
+    <para>
+     There are two modes for the NetApp &o_sharefs; back-end:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       Mode1: <literal>driver_handles_share_servers = True</literal>
```
Comment (asettle) [inline xml/installation-ardana-manila.xml]: probably could remove "Mode1" or is that the name of it too?

File: xml/installation-ardana-manila.xml [hunk 2]
```diff
@@ -92,69 +127,285 @@ the changes to take effect:
...
+share_driver = manila.share.drivers.cephfs.driver.CephFSDriver
+cephfs_conf_path = /etc/ceph/ceph.conf
+cephfs_protocol_helper_type = CEPHFS
+cephfs_auth_id = manila
+cephfs_cluster_name = ceph
+cephfs_enable_snapshots = False</screen>
+    </step>
+    <step>
+     <para>
+      Add CephFS to enabled_share_protocols:
```
Comment (asettle) [inline xml/installation-ardana-manila.xml]: ```<literal>enabled_share_protocols</literal>```

File: xml/installation-ardana-manila.xml [hunk 3]
```diff
@@ -326,10 +577,11 @@ default_share_type = default1</screen>
   </para>
   <para>
    if the compute nodes do not have access to manila back-end server, use the
-   manila-share service on controller nodes instead. You can do so by either running
-   <literal>sudo systemctl stop openstack-manila-share</literal> on compute to turn off
-   its share service or skipping adding "manila-share" to compute hosts in the input-model
-   (<filename>control_plane.yml</filename> in /var/lib/ardana/openstack/my_cloud/definition/data).
+   manila-share service on controller nodes instead. You can do so by either
+   running <literal>sudo systemctl stop openstack-manila-share</literal> on
+   compute to turn off its share service or skipping adding "manila-share" to
```
Comment (asettle) [inline xml/installation-ardana-manila.xml]: ```<literal>manila-share</literal>```

File: xml/installation-ardana-manila.xml [hunk 4]
```diff
@@ -92,69 +127,288 @@ the changes to take effect:
...
+     <para>
+      Configure a backend in the &o_sharefs; configuration file, following the
+      directions and comments in the file.
+     </para>
+<screen>&prompt.ardana;cd ~/openstack/my_cloud
+&prompt.ardana;vi config/manila/manila.conf.j2</screen>
+    </step>
+    <step>
+     <para>
+      Commit your configuration to the local git repo.
```
Comment (dmpop) [inline xml/installation-ardana-manila.xml]: Git

File: xml/installation-ardana-manila.xml [hunk 5]
```diff
@@ -92,69 +127,288 @@ the changes to take effect:
...
+     <filename>manila-share-start.yml</filename>,
+     <filename>manila-share-stop.yml</filename>, and
+     <filename>manila-share-status.yml</filename>.
     </para>
+   </note>
+  </section>
+  <section>
+   <title>Configure CephFS &o_sharefs; Backend</title>
+   <para>
+    Configure a backend in the &o_sharefs; configuration file,
```
Comment (dmpop) [inline xml/installation-ardana-manila.xml]: back-end

File: xml/installation-ardana-manila.xml [hunk 6]
```diff
@@ -92,69 +127,288 @@ the changes to take effect:
...
+    <step>
+     <para>
+      According to the environment, modify back-end specific lines in
+      <filename>~/openstack/my_cloud vi
+      config/manila/manila.conf.j2</filename>.
+     </para>
+    </step>
+    <step>
+     <para>
+      Commit your configuration to the local git repo.
```
Comment (dmpop) [inline xml/installation-ardana-manila.xml]: Git

File: xml/installation-ardana-manila.xml [hunk 7]
```diff
@@ -326,10 +580,12 @@ default_share_type = default1</screen>
   </para>
   <para>
    if the compute nodes do not have access to manila back-end server, use the
```
Comment (dmpop) [inline xml/installation-ardana-manila.xml]: Manilla

Comment (csymons-suse) [reply xml/installation-ardana-manila.xml]: &o_sharefs;

**Pull request merged**